pkg/kubelet/runonce_test.go

```diff
@@ -37,8 +38,13 @@ func TestRunOnce(t *testing.T) {
 	diskSpaceManager, _ := newDiskSpaceManager(cadvisor, DiskSpacePolicy{})
 	fakeRuntime := &kubecontainer.FakeRuntime{}
 
+	basePath, err := ioutil.TempDir("/tmp", "kubelet")
```

Comment:
While many OS's use ```/tmp``` it may be a good idea to use [```os.TempDir()```](https://golang.org/pkg/os/#TempDir) to ensure it's always system correct.